gsa/src/web/pages/agents/detailspage.js

```diff
@@ -173,49 +174,52 @@ const Page = ({
         onInteraction={onInteraction}
       >
         {({activeTab = 0, onActivateTab}) => (
-          <Layout grow="1" flex="column">
-            <TabLayout grow="1" align={['start', 'end']}>
-              <TabList
-                active={activeTab}
-                align={['start', 'stretch']}
-                onActivateTab={onActivateTab}
-              >
-                <Tab>{_('Information')}</Tab>
-                <EntitiesTab entities={entity.userTags}>
-                  {_('User Tags')}
-                </EntitiesTab>
-                <EntitiesTab entities={permissions}>
-                  {_('Permissions')}
-                </EntitiesTab>
-              </TabList>
-            </TabLayout>
+          <React.Fragment>
+            <PageTitle title={_(`Agent: ${entity.name}`)} />
```

Comment:
To be translatable this needs to be
```suggestion
            <PageTitle title={_('Agent: {{name}}', {name: entity.name})} />
```